wot-ig-2023-draft.html

```diff
@@ -284,20 +284,25 @@ <h2>Scope</h2>
         <h3 id="plugfest">PlugFest and Testing</h3>
         <p>
           An important activity is the operation of PlugFests to collect interoperability implementation experience 
-          and to validate the current working assumptions of the potential building blocks discussed in the Web of Things Interest Group.
+          and to validate the current working assumptions of the potential building blocks and features discussed in the Web of Things Working Group task forces (e.g., Thing Description).
           These events enable developers to get together to test their implementations and facilitates networking between partners and experts.
           The Interest Group will seek to encourage work on open source projects and community evaluation of the Web of Things.
           In more detail, PlugFests allow participants to:
         </p>
         <ul>
-          <li>assist with preparing implementation reports for WoT WG Candidate Recommendations</li>
-          <li>test-drive upcoming or proposed technologies for the W3C Recommendation Track</li>
+          <li>test-drive upcoming or proposed technologies for the W3C Recommendation track</li>
           <li>identify and test-drive upcoming or proposed technologies as they apply to the collection of technologies required to address specific industry and user community needs</li>
           <li>take part in interoperability experiments across implementations for ideas at different levels of maturity</li>
           <li>reach out to other communities and new members through an OpenDay for non-members, which usually includes a demo track</li>
           <li>ensure interoperability between WoT technologies and assistive technologies used by people with disabilities</li>
         </ul>
 
+	<p>
+	 Testing (or feature testing) is a specific requirement of the Web of Things Working Group in order to create an implementation report for the publication of the Recommendation documents.
+	 The Web of Things working group will define a set of requirements that must be tested by at least two independent implementations. The Web of Things Intersted Group supports the realization of 
+	 the testing phase and, similar to PlugFest, helps with the inclusion of implementations, e.g. from open source projects and partners. 
```

Comment:
Somewhere we should state clearly that IG can generate "test input" that can be used by the WG to generate the implementation report.

Note: the charter can be "vague" but "input" covers (at least) "test cases/data" (e.g. TD examples) or "test results" (e.g. CSV files declaring the results of tests run externally).

Reply:
ok, I added another sentence that IG can provide "test input"